python/servo/try_parser.py

```diff
@@ -68,15 +80,15 @@ def handle_preset(s: str) -> Optional[JobConfig]:
     elif s in ["wpt", "linux-wpt"]:
         return JobConfig("Linux WPT", Workflow.LINUX, unit_tests=True, wpt_layout=Layout.all())
     elif s in ["wpt-2013", "linux-wpt-2013"]:
-        return JobConfig("Linux WPT legacy-layout", Workflow.LINUX, wpt_layout=Layout.layout2013)
+        return JobConfig("Linux WPT", Workflow.LINUX, wpt_layout=Layout.layout2013)
```

Comment:
Can we just keep different names here and ignore name field when merging.

Reply:
The problem is that the merged job will keep the name of the first job, which will be incorrect. I would like to make a later change that generates the job name lazily, but for now this is the simplest approach.

Reply:
Sounds good to me!